docs/processors/standalone/building.mdx

```diff
@@ -289,6 +289,80 @@ func main() {
 
 Check [Compiling the processor](#compiling-the-processor) for what to do next, and how to compile the processor.
 
+## Schemas
+
+Processors have access to the Schemas in the used Schema Registry using two utility functions:
+1. `schema.Create` : You can use this utility function to create a new Schema and add it to the Schema Registry.
+This function can be called in any of the main processor methods.
+    
+**Example**:
+```go
+func (p *setProcessor) Open(ctx context.Context) error {
+	// Add a new Schema to the Schema registry before starting to process the records.
+	schemaBytes := []byte(`{
+          "name": "record",
+          "type": "record",
+          "fields": [
+            {
+              "name": "admin",
+              "type": "boolean"
+            },
+            {
+              "name": "id",
+              "type": "int"
+            },
+            {
+              "name": "name",
+              "type": "string"
+            }
+          ]
+        }`)
+	_, err := schema.Create(ctx, schema.TypeAvro, "subject1", schemaBytes)
+	return err
+}
+```
+
+2. `schema.Get`: You can use this utility function to get a Schema from the Schema Registry using
+  its `version` and `subject`. This function can be called in any of the main processor methods.
+
+  **Example**:
+```go
+func (p *setProcessor) Process(ctx context.Context, records []opencdc.Record) []sdk.ProcessedRecord {
+out := make([]sdk.ProcessedRecord, 0, len(records))
+for _, record := range records {
+    // get the schema subject name from the metadata
+    subject := rec.Metadata[opencdc.MetadataPayloadSchemaSubject]
+    // get the schema version from the metadata
+    version, err := strconv.Atoi(rec.Metadata[opencdc.MetadataPayloadSchemaVersion])
+    // get the schema 
+    get, err := schema.Get(ctx, subject, version)
```

Comment:
There are helper methods such as `rec.Metadata.GetPayloadSchemaVersion()` (automatically converts the string to an int), and `rec.Metadata.GetPayloadSchemaSubject()`, so you don't have to work with the map and constants.

Reply:
did not know about these! thank you